admin/class-settings.php

```diff
@@ -153,7 +153,9 @@ public function toggle_addon() {
 		if ( ! isset( $_POST['nonce'] ) || ! wp_verify_nonce( $_POST['nonce'], 'mbrs_toggle_addon' ) ) {
 			die();
 		}
-
+		if ( ! current_user_can( 'manage_options' ) ) {
+			die( 'You are not allowed to make these changes' );
```

Comment:
@kalpeshBhagat 

I think we can use the WP function to send proper JSON response as message is involved.

```
wp_send_json_error( array(
  'msg' => esc_html__( 'You are not allowed to make these changes.', 'members' )
) );
```